algorithm_exercises/27. Remove Element/solution.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {number[]} nums
+ * @param {number} val
+ * @return {number}
+ */
+var removeElement = function (nums, val) {
+    let length = nums.length
```

Comment:
Since length is not modified it can be a constant. 
Same for the function

Reply:
addresed in latest commit